.github/workflows/specification-version.yml

```diff
@@ -0,0 +1,55 @@
+name: Specification version check
+
+on:
+  schedule:
+    - cron: '0 13 * * *'
+  workflow_dispatch:
+
+jobs:
+  check-spec-version:
+    name: Open an issue when spec version bumps
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - name: Get supported version
+      id: get-version
+      run: |
+        python3 -m pip install -e .
+        script="from tuf.api.metadata import SPECIFICATION_VERSION; \
+                print(f\"v{'.'.join(SPECIFICATION_VERSION)}\")"
+        ver=$(python3 -c "$script")
+        echo "::set-output name=version::$ver"
+    - name: Open issue (if needed)
+      uses: actions/github-script@v5
+      with:
+        script: |
+          const release = await github.rest.repos.getLatestRelease({
+            owner: "theupdateframework",
+            repo: "specification"
+          });
+          const spec_version = release.data.tag_name
+          const supported_version = "${{ steps.get-version.outputs.version }}"
+          if (spec_version != supported_version) {
+            console.log("Specification version does not match...")
+            const repo = context.repo.owner + "/" + context.repo.repo
+            const issues = await github.rest.search.issuesAndPullRequests({
+              q: "specification+has+new+version+in:title+state:open+type:issue+repo:" + repo,
```

Comment:
I'm guessing the "in:title" search might actually require quotes to get an exact search string but I think I'm fine with this: let's have another look if we get false positives in this.